Model/EventsForThirdPartyModules.php

```diff
@@ -71,6 +72,14 @@ class EventsForThirdPartyModules
                     "sendClasses" => ["Aheadworks\Giftcard\Api\GiftcardCartManagementInterface"],
                     "boltClass" => Aheadworks_Giftcard::class,
                 ],
+                [
+                    "module" => "Mirasvit_Credit",
+                    "sendClasses" => ["Mirasvit\Credit\Helper\Data",
+                                      "Mirasvit\Credit\Service\Calculation",
+                                      ["Mirasvit\Credit\Api\Config\CalculationConfigInterface",
```

Comment:
Please add here explains why we use two different classes

Reply:
Done